src/app/core/models/product/product.helper.ts

```diff
@@ -151,4 +151,13 @@ export class ProductHelper {
       };
     }
   }
+
+  static showAddToCart(product: Product): boolean {
+    return !!(
+      product.inStock &&
+      product.availability &&
+      !ProductHelper.isMasterProduct(product) &&
+      (product.salePrice || ProductHelper.isVariationProduct(product) || ProductHelper.isRetailSet(product))
```

Comment:
what is the logic of this line?

Show if it has a sale price -- ok
Show if it is a variation product or retail set even if it has no price? -- is that even possible?

Reply:
retailset has other attributes: summedUpSalePrice, minSalePrice
the condition for variation-products is not really necessary (removed)

Reply:
okay :+1: